cranelift/codegen/src/isa/x64/inst/emit.rs

```diff
@@ -516,6 +516,128 @@ pub(crate) fn emit(
             }
         }
 
+        Inst::Div {
+            size,
+            signed,
+            divisor,
+            loc,
+        } => {
```

Comment:
Please could you add here, a comment showing what instructions are actually emitted?  It's pretty hard to infer from reading the code.  Same for SignExtendRaxRdx et al.